core/cc/linux/get_gles_proc_address.cpp

```diff
@@ -18,12 +18,25 @@
 #include "../get_gles_proc_address.h"
 #include "../log.h"
 
+#include <dlfcn.h>
+
 namespace {
 
 void* getGlesProcAddress(const char *name, bool bypassLocal) {
     using namespace core;
     typedef void* (*GPAPROC)(const char *name);
 
+    // The mesa driver does bad things with LLVM. Since we also use llvm, 
+    // we can't have the mesa driver do bad things to our code.
+    // Therefore we should preload any versions of llvm that may be required
+    // into the start of our address space.
```

Comment:
Maybe create a bug explaining what the Bad Thing™ is and link to it here. That way, if there's a mesa driver that uses an LLVM version not listed here, people can find the issue and know what they need to do.

Reply:
Good plan! I will do that.